aiks/paint.cc

```diff
@@ -11,6 +11,7 @@ namespace impeller {
 std::shared_ptr<Contents> Paint::CreateContentsForEntity(Path path,
                                                          bool cover) const {
   if (contents) {
+    contents->SetPath(std::move(path));
```

Comment:
Please add a DCHECK that asserts that `SetPath` has been called before `Render`.

Reply:
Done. Also been thinking about adding an `IsValid` for contents in general, but will save that for another day.